code/workspaces/auth-service/src/dataaccess/userRolesRepository.spec.js

```diff
@@ -2,46 +2,49 @@ import userRoleRepository from './userRolesRepository';
 import database from '../config/database';
 import databaseMock from './testUtil/databaseMock';
 
-const wrapDocument = document => ({
-  ...document,
-  toObject: () => document,
-});
+const user1 = {
+  userId: 'uid1',
+  userName: 'user1',
+  instanceAdmin: false,
+  projectRoles: [
+    { projectKey: 'project 1', role: 'admin' },
+    { projectKey: 'project 2', role: 'user' },
+  ],
+};
+
+const user2 = {
+  userId: 'uid2',
+  userName: 'user2',
+  instanceAdmin: true,
+  catalogueRole: 'publisher',
+  projectRoles: [
+    { projectKey: 'project 2', role: 'viewer' },
+  ],
+};
+
+const duplicateUser2 = {
+  userId: 'uid2',
+  userName: 'user2',
+  projectRoles: [
+    { projectKey: 'project 3', role: 'admin' },
+  ],
+};
+
+const userWithoutIdentity = {
+  // user without identity
```

Comment:
Don't think we need this comment with the given variable name for the user.

Reply:
Thanks, fixed!